### ISSUE TITLE
Focus may stuck on the input

### ISSUE THREAD

**Issue by shrpne** [KazanExpress/vue-simple-suggest#297]
## **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request

## **What is the current behavior?**
Focus on the input may stuck sometimes.

## **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
https://jsfiddle.net/vkdn6bLe/
- click on the first input
- hover over any suggestion item, don't move mouse anymore
- write "asd"
- try to focus second input (it doesn't get focus, second input always gain focus back, the only way to fix it: delete "asd", mouseenter suggestion list, mouseleave suggestion list, done, focus can be changed now)

## **What is the expected behavior?**
Focus should not stuck

### How are you importing Vue-simple-suggest?
Not matters

## **Please tell us about your environment:**
  
  - Vue.js Version: 2.6.11
  - Vue-simple-suggest version: 1.10.1
  - Browser: Firefox, Chrome, probably all

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
I'm about to make a PR to fix it

**Comment by xlcrr** [comment]
I can de-focus the input, but it returns a bad error.

```
<div class="container" @click="deFocusInputs">
  <div class="wrapper" @click.stop>
     < vss />
   </div>
</div>
```

```
deFocusInputs () {
            this.$refs.vss[0].isInFocus = false;
            Array.prototype.forEach.call(document.getElementsByClassName('input'), function(el) {
              el.blur();
            });
        },
```

Returns 

```
This should never happen!

If you encountered this error, please make sure that your input component emits 'focus' events properly.

For more info see https://github.com/KazanExpress/vue-simple-suggest#custom-input.

If your 'vue-simple-suggest' setup does not include a custom input component - please,
          report to https://github.com/KazanExpress/vue-simple-suggest/issues/new
```

The error is silenced with production code

**Comment by shrpne** [comment]
Fixed in #298
